.github/workflows/checks.yml

```
@@ -10,12 +10,12 @@ jobs:
   checks:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
     - name: Install prerequisites
       run: |
-          pip3 install setuptools
-          pip3 install -r requirements.txt
+        apt install -qq clang-format
```

As I remember, 'sudo' is needed for 'apt' in GitHub Actions.
```suggestion
        sudo apt-get install -qq clang-format
```